specification/specification.yaml

```
@@ -0,0 +1,531 @@
+openapi: 3.1.0
+info:
+  version: '1.0'
+  title: API сервер для проекта six-cities.
+description: |-
+  * Список ресурсов и маршрутов сервера six-cities.
+license:
+  name: MIT
+  url: https://opensource.org/licenses/MIT
+  version: 2.0.0
```

тут с форматированием есть проблемки
```suggestion
info:
  version: '1.0'
  title: API сервер для проекта six-cities.
  description: |-
    * Список ресурсов и маршрутов сервера six-cities.
  license:
    name: MIT
    url: https://opensource.org/licenses/MIT
```

кое где отступов не хватает